### PR TITLE
Fixes and fastclick addition

### PR DESCRIPTION
**Pull request by ironland**
Fix code style errors. 
Add fastclick. 
Change theme.js to not use static class properties.

### CODE REVIEW

File: assets/scripts/main.js [hunk 1]
```diff
@@ -5,6 +5,11 @@
 // Require 3rd party libraries
 require( 'babel-polyfill' );
 require( __dirname + '/modernizr.js' );
+
+// Require fastclick for a faster mobile UX.
+let FastClick = require( 'fastclick' );
```
Comment (Nomafin) [inline assets/scripts/main.js]: const

File: assets/scripts/theme.js [hunk 2]
```diff
@@ -82,15 +77,18 @@ class Theme {
...
      */
-    setGlobalControllers(globalControllers) {
-        if (Array.isArray(globalControllers)) {
-            for (let i = 0; i < globalControllers.length; i++) {
+    setGlobalControllers( globalControllers ) {
+        if ( Array.isArray( globalControllers ) ) {
+            for ( let i = 0; i < globalControllers.length; i++ ) {
+
                 // Get the class name from the class reference.
                 let className = globalControllers[i].name;
```
Comment (Nomafin) [inline assets/scripts/theme.js]: const

File: assets/scripts/main.js (hunk 1 above)
Comment (Nomafin) [inline assets/scripts/main.js]: const

File: assets/scripts/theme.js [hunk 3]
```diff
@@ -100,14 +98,17 @@ class Theme {
...
      */
-    setTemplateControllers(templateControllers) {
-        if (Array.isArray(templateControllers)) {
-            for (let i = 0; i < templateControllers.length; i++) {
+    setTemplateControllers( templateControllers ) {
+        if ( Array.isArray( templateControllers ) ) {
+            for ( let i = 0; i < templateControllers.length; i++ ) {
+
                 // Get the class name from the class reference.
                 let className = templateControllers[i].name;
```
Comment (Nomafin) [inline assets/scripts/theme.js]: const

**Review by Nomafin — CHANGES_REQUESTED** [review]

**Review by villesiltala — APPROVED** [review]

**Comment by ironland** [comment]
fikasttu muutospyynnöt ja mergetty

**Review by Nomafin — APPROVED** [review]
ok

**Pull request merged**